include/fmt/chrono.h

```diff
@@ -1135,10 +1244,20 @@ template <typename OutputIt, typename Char> class tm_writer {
   }
 
  public:
+  tm_writer(const std::locale& loc, OutputIt out, const std::tm& tm,
+            std::chrono::duration<Rep, Period> subsecs)
```

Comment:
Let's pass subsecs as a pointer with null indicating "no subseconds" and merge the two ctors into one to reduce copy pasta.